web/src/main/java/org/perfrepo/web/dao/TestExecutionDAO.java

```diff
@@ -611,10 +777,8 @@ public static TestExecution fetchTest(TestExecution exec) {
     * @return TestExecution with fetched tags
     */
    public static TestExecution fetchTags(TestExecution testExecution) {
-      Collection<TestExecutionTag> cloneTags = new ArrayList<TestExecutionTag>();
-      for (TestExecutionTag interObject : testExecution.getTestExecutionTags()) {
-         cloneTags.add(interObject.cloneWithTag());
-      }
+      Collection<TestExecutionTag> cloneTags = new ArrayList<>();
+      testExecution.getTestExecutionTags().stream().forEach(interObject -> cloneTags.add(interObject));
```

Comment:
This should not be a problem, default fetching strategy for @OneToOne and @ManyToOne relationships is EAGER, therefore no clone is necessary.